pages/upgrade.rst

```diff
@@ -37,6 +37,8 @@ For .rpm based systems :ref:`this update guide <operating_package_upgrade_rpm-yu
 Upgrading Elasticsearch
 -----------------------
 
+.. warning:: We caution you not to upgrade Elasticsearch to 7.11! You are absolutely forbidden to upgrade to this version. If you do so, it will break your instance.ß
```

Comment:
Use "!" at end of sentence for consistency and remove stray "ß" character:
```suggestion
.. warning:: We caution you not to upgrade Elasticsearch to 7.11! You are absolutely forbidden to upgrade to this version. If you do so, it will break your instance!
```

Reply:
Take a look now, @bernd. Good to go?

Reply:
@dulanism Thanks for the fixes! :+1: 

To be honest, the wording sounds rather strange to me. I am not a native speaker, so please take my comment with a grain of salt! :smile: 

I am specifically talking about the "You are absolutely forbidden to upgrade to this version." part. That sounds very strong and weird, because we cannot forbid a user to do something. :smile: My personal preference would be something along the lines of:

> We caution you not to install or upgrade Elasticsearch to 7.11 and later! It is not supported. If you do so, it will break your instance!

What do you think?